assets/js/hooks-shim.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { createHooks } from '@wordpress/hooks__non-shim';
+
+if ( global._GoogleSiteKitHooks === undefined ) {
+	global._GoogleSiteKitHooks = createHooks();
```

Comment:
@tofumatt Can you explain a bit more what the problem here was? Why can't we keep this internal?

Reply:
@felixarntz I don't know that we ever found out the exact cause (I think something funny happening during compilation) but basically hooks just wouldn't work at all, and raised no errors. From what I found `@wordpress/hooks` the `actions` and `filters` exported were both `null` when they were supposed to be objects [like this](https://github.com/WordPress/gutenberg/blob/4c1ef946655669bf3f031579033b75f7fae7b6d7/packages/hooks/src/createHooks.js#L18-L21).

I took a look at this again yesterday to investigate and found that hooks are mostly working now without this global but some elements of the UI are missing so it might be worth digging into this a little more. This may have changed due to other changes/fixes in this branch that seemed unrelated (such as opting out of amd parsing), but it's hard to say.